packages/ui-flex/src/Flex/Item/props.ts

```diff
@@ -110,6 +110,14 @@ type FlexItemOwnProps = {
    * Places dashed lines around the component's borders to help debug your layout
    */
   withVisualDebug?: boolean
+  /**
+   * Specifies the order of the `Flex.Item` inside the `Flex` component.
+   *
+   * Items with the same order revert to source order.
+   *
+   * Utilizes the order flex CSS property. Default order CSS value is 0.
+   */
+  order?: number
```

Comment:
I don't know if I'd over explain the behavior. It works like the native css stuff, maybe we could just say that? I'm not sure
Also, I don't think it defaults to 0. It defaults to undefined